mlst/2.22.1/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:focal as app
+
+ARG MLST_VER="2.22.1"
+
+LABEL base.image="ubuntu:focal"
+LABEL container.version="1"
+LABEL software="mlst"
+LABEL software.version="${MLST_VER}"
+LABEL description="Scan contig files against PubMLST typing schemes"
+LABEL website="https://github.com/tseemann/mlst"
+LABEL license="https://github.com/tseemann/mlst/blob/master/LICENSE"
+LABEL maintainer="Curtis Kapsak"
+LABEL maintainer.email="curtis.kapsak@theiagen.com"
+
+# install dependencies via apt; cleanup apt garbage
+# blast from ubuntu:focal is v2.9.0 (as of 2022-05-09)
+RUN apt-get update && apt-get install -y --no-install-recommends \
+ wget \
+ ca-certificates \
+ libmoo-perl \
+ liblist-moreutils-perl \
+ libjson-perl \
+ gzip \
+ file \
+ ncbi-blast+ && \
+ apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# get any2fasta
+RUN cd /usr/local/bin && \
```

Comment:
Would it be a good idea to add a version to this?

Reply:
ooo yes, that's a good idea. I'll update the dockerfile